pkg/minikube/openshiftversions/openshift_versions_test.go

```diff
@@ -17,149 +17,67 @@ limitations under the License.
 package openshiftversions
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"net/http"
-	"net/http/httptest"
-	"net/url"
+	"io/ioutil"
```

Comment:
Same with the tests, I removed all github related test cases and mostly those were skipping because rate limit issue and here it's pure expected and actual values now.